dom-querying/main.js

```diff
@@ -0,0 +1,14 @@
+console.log('hello, world!');
+
+console.log(document.querySelector('h1'));
+console.dir('h1');
+
+console.log(document.querySelector('#explanation'));
+console.dir('#explanation');
+
+console.log(document.querySelector('.hint'));
+console.dir('.class');
```

Comment:
This is not what the exercise requested. You should be passing the DOM element itself as an argument here, not the CSS selector for it.